### PR TITLE
Standardize formatting in UG 

### PR DESCRIPTION
**Pull request by erv-teo**
This PR also 
- fixes a minor typo in the DG
- changes the UML diagram to use a better resolution

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -382,7 +416,9 @@ On entering the `help` command, the following window will pop up:
 
 Allows you to clear all entries from the address book.
 
+<div markdown="span" class="alert alert-warning">
 :exclamation: **Caution:** This action is irreversible.
```
Comment (ejnan) [inline docs/UserGuide.md]: image
Caution is not bolded and users can see **, which may be due to spaces. The formatting for the boxes will probably only be seen once merged

**Review by ejnan — COMMENTED** [review]
LGTM other than the format for Caution with ** at the sides

File: docs/UserGuide.md (hunk 1 above)
Comment (erv-teo) [reply docs/UserGuide.md]: On my screen the display looks fine, maybe we can merge and see if the problem still persists? 
Screenshot 2024-04-12 at 16 29 24

**Review by ryanlimdx — APPROVED** [review]
LGTM

**Pull request merged**